test/functional/test_framework/mininode.py

```diff
@@ -139,12 +139,20 @@ def peer_connect(self, dstaddr, dstport, *, net, timeout_factor):
         self.on_connection_send_msg = None
         self.recvbuf = b""
         self.magic_bytes = MAGIC_BYTES[net]
-        logger.debug('Connecting to Bitcoin Node: %s:%d' % (self.dstaddr, self.dstport))
+
+    def peer_connect(self, dstaddr, dstport, *, net, timeout_factor):
+        self.peer_connect_helper(dstaddr, dstport, net, timeout_factor)
 
         loop = NetworkThread.network_event_loop
-        conn_gen_unsafe = loop.create_connection(lambda: self, host=self.dstaddr, port=self.dstport)
-        conn_gen = lambda: loop.call_soon_threadsafe(loop.create_task, conn_gen_unsafe)
-        return conn_gen
+        logger.debug('Connecting to Bitcoin Node: %s:%d' % (self.dstaddr, self.dstport))
+        coroutine = loop.create_connection(lambda: self, host=self.dstaddr, port=self.dstport)
+        return lambda: loop.call_soon_threadsafe(loop.create_task, coroutine)
+
+    def peer_outbound_connect(self, dstaddr, dstport, connect_cb, connect_id=0, *, net, timeout_factor):
```

Comment:
Consider renaming this function (and the equivalents in `P2PInterface` and `TestNode`) to `peer_accept_connection()` or similar. From the perspective of the `P2PConnection`, this is an inbound connection since it's initiated by the bitcoind node.

Reply:
Good point. The `TestNode` function should be called `add_outbound_p2p_connection()`

Reply:
renamed both instances of `peer_outbound_connect` to `peer_accept_connection`. thanks!